definitions/ext-rfscanner/golden_metric.yml

```diff
@@ -0,0 +1,39 @@
+# Determine metrics for VALET
+
+
+latency:
+  title: Latency  (ms)
+  displayAsValue: true
+
+
+  query:
+    select: average(totalTime) AS 'Latency'
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+
+throughput:
+    title: Volume
+    
+    query:
+      select: count(*) AS 'Volume'
+      from: THD_RF_CONNECTION_EVENT
+      facet: macAddress      
+
+# deviceAvailability:
+#  title: Availability
+#  displayAsValue: true
+# 
+# query:
+# need to figure this query out
+#  select: pseudo query   "count(*) where status !='CLOSED)"/count(*)*100 AS "Availability"
+# from: THD_RF_CONNECTION_EVENT
+
+errors:
+  title: Errors
+
+  query:
+    select: count(message) AS 'Errors'
+    from: Log
+    condition: where message like '%error%' or message like '%Error%' and message NOT LIKE '%lqqqq%' or message NOT LIKE '%Winsock error 10042%'
```

Comment:
we don't have a condition clause. We have a `WHERE` one.

But all golden metrics will filter by the `entity.guid` so unless this log is attached to this entity this GM will not work.